pkg/blockrsync/hasher.go

```diff
@@ -30,6 +36,7 @@ type Hasher interface {
 	SerializeHashes(io.Writer) error
 	DeserializeHashes(io.Reader) (int64, map[int64][]byte, error)
 	BlockSize() int64
+	CompareHashHash(io.ReadWriter) (bool, error)
```

Comment:
find this name confusing, maybe s/HashHash/HashOfHash/

Reply:
Done